lib/pbench/server/database/models/users.py

```diff
@@ -17,6 +16,7 @@ class User(Database.Base):
     last_name = Column(String(255), unique=False, nullable=False)
     password = Column(LargeBinary(500), nullable=False)
     registered_on = Column(DateTime, nullable=False)
+    bcrypt_log_rounds = Column(Integer, nullable=False)
```

Comment:
Really? Different per-user? Huh.

Reply:
Yeah I removed the requirement of `bcrypt_log_rounds` from our default config file since we dont know what constitute the good hash log rounds number and so I decided rather than us choosing some arbitrary number will just go with the default number i.e. 12 and 12 seems to be a good enough for adding a complexity to an encrypted passwords.